tests/cross_functional/krkn_chaos/test_krkn_container_chaos.py

```
@@ -364,13 +357,13 @@ def test_krkn_random_pod_kill(
             )
             raise
         except Exception as e:
-            log.error(f"❌ Random pod kill chaos failed for {stress_level} level: {e}")
+            logger.exception(
+                f"Random pod kill chaos failed for {stress_level} level: {e}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)